include/vctr/Containers/Vector.h

```diff
@@ -200,11 +200,40 @@ class Vector : public VctrBase<ElementType, detail::StdVectorType<ElementType, A
     //==============================================================================
     // Operators
     //==============================================================================
-    /** Copies the data of the source container to this Vector. This Vector will be resized to fit if needed. */
+    /** Copies or moves the data of the source container to this Vector. This Vector will be resized to fit if needed. */
     template <has::sizeAndDataWithElementType<ElementType> Container>
-    constexpr Vector& operator= (const Container& containerToCopyDataFrom)
+    constexpr Vector& operator= (Container&& containerToCopyDataFrom)
```

Comment:
so assigning an existing container still works? yep, obviously :)

Reply:
Yes, this only makes it possible to additionally move values if we move assign some container